### ISSUE TITLE
Publish package to PyPI with new name

### ISSUE THREAD

**Issue by simonw** [simonw/click-default-group-wheel#3]
I'm going to publish this as `click-default-group-wheel` so I can depend on it from `sqlite-utils` which will hopefully allow it to be installed using `micropip` in Pyodide.

When this issue is closed I'll retire this package entirely:

- https://github.com/click-contrib/click-default-group/issues/21

**Comment by simonw** [comment]
To test this package before pushing it to PyPI I did this:

```
git clone git:github.com:simonw/click-default-group-wheel
cd click-default-group-wheel
python3 -m build
s3-credentials put-object simonwillison-cors-allowed-public \
  click_default_group_wheel-1.2.2-py3-none-any.whl \
  dist/click_default_group_wheel-1.2.2-py3-none-any.whl
```
Then on https://pyodide.org/en/stable/console.html
```
Welcome to the Pyodide terminal emulator 🐍
Python 3.10.2 (main, Apr  9 2022 20:52:01) on WebAssembly VM
Type "help", "copyright", "credits" or "license" for more information.
>>> import micropip
>>> await micropip.install(
...     "https://s3.amazonaws.com/simonwillison-cors-allowed-public/click_default_group_wheel-1.2.2-py3-none-any.whl"
... )
>>> import click_default_group
>>> click_default_group
<module 'click_default_group' from '/lib/python3.10/site-packages/click_default_group.py'>
>>> dir(click_default_group)
['DefaultCommandFormatter', 'DefaultGroup', '__all__', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '
__name__', '__package__', '__spec__', '__version__', 'click', 'warnings']
>>> 
```

**Comment by simonw** [comment]
It's live: https://pypi.org/project/click-default-group-wheel/

**Comment by simonw** [comment]
I built a package of `sqlite-utils` that depends on this: https://s3.amazonaws.com/simonwillison-cors-allowed-public/sqlite_utils-3.26-py3-none-any.whl

Now in https://pyodide.org/en/stable/console.html

```
>>> import micropip
>>> await micropip.install("https://s3.amazonaws.com/simonwillison-cors-allowed-public/sqlite_utils-3.26-py3-none-any.whl")
>>> import sqlite_utils
>>> db = sqlite_utils.Database(memory=True)
>>> list(db.query("select 32443 + 55"))
[{'32443 + 55': 32498}]
```

**Comment by simonw** [comment]
Shipped `sqlite-utils 3.26.1` to take advantage of this - so now `await micropip.install("sqlite-utils")` works too.

https://sqlite-utils.datasette.io/en/stable/changelog.html#v3-26-1
